scripts/simple_doc_generator.py

```
@@ -0,0 +1,370 @@
+        # Count source files
+        src_dir = self.project_root / 'src'
+        if src_dir.exists():
+            for py_file in src_dir.rglob('*.py'):
+                info['src_files'] += 1
+                try:
+                    with open(py_file, 'r', encoding='utf-8') as f:
+                        content = f.read()
+                        info['total_lines'] += len(content.splitlines())
+                        info['total_functions'] += content.count('def ')
+                        info['total_classes'] += content.count('class ')
+                except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)